Older SLE guests have a two stage installation that need the ISO.
Newer SLE PV guests hang when a cdrom device is attached without
an ISO file.

Index: virt-manager-5.1.0/virtinst/install/installer.py
===================================================================
--- virt-manager-5.1.0.orig/virtinst/install/installer.py
+++ virt-manager-5.1.0/virtinst/install/installer.py
@@ -178,7 +178,8 @@ class Installer:
     def _remove_install_cdrom_media(self, guest):
         if not self._install_cdrom_device_added:
             return
-        if guest.osinfo.is_windows():
+        if (guest.osinfo.is_windows() or
+            guest.osinfo.name.startswith(("sles", "sled", "opensuse"))):
             # Keep media attached for windows which has a multi stage install
             return
         for disk in guest.devices.disk:
